src/ResourceManager/AzureBatch/Commands.Batch/Models/NodeFileOperationParameters.cs

```diff
@@ -84,9 +84,9 @@ public class NodeFileOperationParameters : BatchClientParametersBase
         public string ComputeNodeId { get; private set; }
 
         /// <summary>
-        /// The name of the node file
+        /// The path of the node file
         /// </summary>
-        public string NodeFileName { get; private set; }
+        public string Path { get; private set; }
```

Comment:
same here.

Reply:
My intent is to just leave this as is (no obsolete) since this is a breaking change, unless otherwise suggested by one of the powershell maintainers.

Reply:
Same comment, must document in upcoming-breaking changes

Reply:
This is actually only a type used internally in the cmdlets, it's not part of a cmdlet or return type. It's just used to store information about what request is being performed so we can pass it off to the C# SDK, so I don't think there's anything to really say in upcoming-breaking-changes. It was already covered in the `Name` -> `Path` change for the `GetNodeFile`/`RemoveBatchNodeFile` cmdlets, which was already documented.